dev/com.ibm.ws.cdi.internal/resources/com/ibm/ws/cdi/internal/resources/CDI.nlsprops

```diff
@@ -68,3 +68,7 @@ no.injection.target.CWOWB1008E.useraction=Ensure that the archive the class belo
 implicit.bean.scanning.disabled.CWOWB1009W=CWOWB1009W: Implicit bean archives are disabled.
 implicit.bean.scanning.disabled.CWOWB1009W.explanation=Archives are not scanned for CDI beans unless they have a beans.xml file.
 implicit.bean.scanning.disabled.CWOWB1009W.useraction=Ensure that every archive that uses CDI has a beans.xml file or re-enable implicit beans scanning in your server.xml file.
+
+spi.extension.failed.to.construct.CWOWB10010E=CWOWB10010E: Failed to create CDI Extension via SPI.
```

Comment:
Creating CDI Extension via CDI SPI failed.

Reply:
I don't want to change this one if I don't have to because it's already gone off to translation.